Dangerfile.js

```diff
@@ -0,0 +1,37 @@
+// Dangerfile.js
+import { danger } from 'danger';
+import * as logger from '@markbind/core/src/utils/logger';
+
+const couplings = {
+  '.github/workflows/check-file-diff.yml': [
+    'package.json',
+    'babel.config.js',
+  ],
+  'babel.config.js': [
+    'package.json',
+    'lerna.json',
+  ],
+};
+
+const { git } = danger;
+
+Promise.resolve().then(() => {
+  const allModifiedFiles = [...git.modified_files, ...git.created_files];
+  const messages = [];
+
+  Object.entries(couplings).forEach(([implFile, testOrDocFiles]) => {
```

Comment:
This makes sense to me in theory, but given that the sample couplings you have here are package.json and either lerna or babel, which is not a TestOrDoc file, I think reading this code is confusing. Maybe we should name it as "dependent files" or something?

Or maybe this isn't such a good example of a coupling? You could put site and site test maybe 🤔